### PR TITLE
updated issue and pullrequest templates

### LINKED ISSUE #5051: Update Templates and add a documentation for the editor
The following should be updated:

- [x] Update the current Issue Template to the new version (https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates)
- [x] Add a pull request template (https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates)
- [x] Add a documentation (wiki page) how to use our [editor](live.bootstrap-table.com/) and link that page in the Issue and PR template

### PR DESCRIPTION
**Pull request by UtechtDustin**
This PR add some new issue templates and a pullrequest template.

Fix #5051

### CODE REVIEW

**Comment by UtechtDustin** [comment]
Before we can merge that PR we have to write a explaination about our Online Editor (https://github.com/wenzhixin/bootstrap-table/wiki/Online-Editor-Explanation).

@wenzhixin 
Could you maybe do it, your the one who create the editor and has the most knowhow about that.
Btw. it would be greate if you could trim the input of the "GitHub src branch" option.

**Comment by djhvscf** [comment]
Awesome work!!

**Comment by djhvscf** [comment]
@UtechtDustin could we merge this?

**Comment by UtechtDustin** [comment]
If you dont find any grammer issues or something like this, yes :-)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+**Bug fix?**
+<!-- yes/no -->
+
+**New Feature?**
+<!-- yes/no -->
+
+**Resolve a issue?**
```
Comment (djhvscf) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: Resolve an issue

Comment (UtechtDustin) [reply .github/PULL_REQUEST_TEMPLATE.md]: done

**Pull request merged**